package.json

```diff
@@ -25,6 +25,7 @@
     "gulp-uglify": "^3.0.0",
     "merge": "^1.2.0",
     "napa": "^3.0.0",
+    "object-fit-polyfill": "^0.1.0",
```

Comment:
I don't think this is the right package--the official package name on npm should be `objectFitPolyfill`.